Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -270,6 +273,66 @@ void MenuBar::AddTableColumnsMenu(QMenu* view_menu)
   }
 }
 
+void MenuBar::AddShowPlatformsMenu(QMenu* view_menu)
+{
+  static const QMap<QString, bool*> platform_menu{
```

Comment:
Perhaps you should rename it to ``platform_map`` or something like that.
That makes the code much clearer.